tools/util/include/cutlass/util/reference/host/tensor_fill.h

```diff
@@ -205,23 +205,29 @@ struct RandomGaussianFunc<complex<Element> > {
 
     Element reals[2];
 
-    for (int i = 0; i < 2; ++i) {
-      // Box-Muller transform to generate random numbers with Normal distribution
-      double u1 = double(std::rand()) / double(RAND_MAX);
-      double u2 = double(std::rand()) / double(RAND_MAX);
-
-      // Compute Gaussian random value
-      double rnd = std::sqrt(-2 * std::log(u1)) * std::cos(2 * pi * u2);
-      rnd = mean + stddev * rnd;
+    // Box-Muller transform to generate random numbers with Normal distribution
+    double u1 = double(std::rand()) / double(RAND_MAX);
+    double u2 = double(std::rand()) / double(RAND_MAX);
 
-      if (int_scale >= 0) {
-        rnd = double(int(rnd * double(1 << int_scale)));
-        reals[i] = from_real<Element>(rnd / double(1 << int_scale));
+    // Compute Gaussian random values
+    double rnd1 = std::sqrt(-2 * std::log(u1)) * std::cos(2 * pi * u2);
+    double rnd2 = std::sqrt(-2 * std::log(u1)) * std::sin(2 * pi * u2);
+    rnd1 = mean + stddev * rnd1;
+    rnd2 = mean + stddev * rnd2;
```

Comment:
small nit, can we put box-muller algorithm into an inline helper function with documentation. it can be used in both complex and quaternion then.